provisioning/roles/chrome/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+# currently only the latest version is supported
```

Comment:
These should be in `defaults/main.yml` (otherwise you won't be able to change them in `parameters.yml`).